federation/apis/federation/v1alpha1/types.go

```diff
@@ -84,6 +84,11 @@ type ClusterStatus struct {
 	// Allocatable represents the total resources of a cluster that are available for scheduling.
 	Allocatable v1.ResourceList `json:"allocatable,omitempty" protobuf:"bytes,3,rep,name=allocatable,casttype=k8s.io/kubernetes/pkg/api/v1.ResourceList,castkey=k8s.io/kubernetes/pkg/api/v1.ResourceName"`
 	ClusterMeta `json:",inline" protobuf:"bytes,4,opt,name=clusterMeta"`
+	// Zones is the list of avaliability zones in which the nodes of the cluster exist, e.g. 'us-east1-a'.
```

Comment:
FYI: These comments should refer to json name rather than the go struct name. So this should be "zones is the list..." instead of "Zones is the list..." but I see we are not following that rule in federation types.go.
So we can fix Zones and Regions when we fix all others in these files.